.github/README.md

```diff
@@ -14,44 +14,41 @@ The requirements are intentionally sparse (the devil is in the detail). We don't
+- Read and understand [The Challenge](#the-challenge).
+- Create a solution that satisfies the [Requirements](#requirements).
+- Send the link to the repository containing your solution to the person who contacted you.
 
 ## The Challenge
 
 ### Background
 
-There's a new set of product features being planned for [BLACK OPAL](https://q-ctrl.com/products/black-opal/) whereby a customer will be able to create and manage quantum controls. You can think of these controls as being the individual inputs a customer may employ to manipulate their quantum system using the [Q-CTRL App](https://app.q-ctrl.com/).
+At Q-CTRL, we use a suite of microservices to support our products. Typically, these microservices are written using a Python web framework and expose a [GraphQL](https://graphql.org/) API so clients and other services can communicate with it.
 
 ### Requirements
 
-Create a RESTful API using [Django](https://www.djangoproject.com/). The API should be backed by a [PostgreSQL](https://www.postgresql.org/) database, conform to the [JSON:API](https://jsonapi.org/) specification and should implement endpoints that provide the following functionality:
-
-1. Create a new control
-1. List all controls (five per page)
-1. Get a specific control
-1. Update a specific control
-1. Delete a specific control
-1. Bulk upload controls in CSV format
-1. Download controls in CSV format
-
-### Additional Information
-
-Below are the attributes of a control. All attributes are required.
-
-| Name              | Description                                                                                                                     |
-|-------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| Name              | The name of the control (e.g. "Single-Qubit Driven").                                                                           |
-| Type              | Valid control types are Primitive, CORPSE, Gaussian, CinBB and CinSK.                                                           |
-| Maximum Rabi Rate | The maximum achievable angular frequency of the Rabi cycle for a driven quantum transition. This is a number between 0 and 100. |
-| Polar Angle       | An angle measured from the z-axis on the Bloch sphere. This is a number between 0 and 1 (units of pi).                          |
-
-The below assets have been provided to help complete the challenge.
-
-| Name                                | Description     |
-|-------------------------------------|-----------------|
-| [controls.csv](assets/controls.csv) | Sample CSV file |
+1. Create a new service using the Python web framework of your choice (e.g. [Django](https://www.djangoproject.com/), [Flask](https://palletsprojects.com/p/flask/), etc.).
+1. All dependencies should be captured in a `requirements.txt` file at the root of the repository.
```

Comment:
I put that in because I want to be able to validate the implementation easily using the query. I didn't want to have to deal with building docker images etc. which would be overkill.